wagtailio/project_styleguide/templates/patterns/includes/svg_sprites.html

```diff
@@ -20,4 +20,6 @@
.02-.02a.387.387 0 01.021-.02l.02-.021.02-.02.02-.02.021-.021a.189.189 0 00.02-.02l.021-.021a.189.189 0 00.02-.02l.021-.021.021-.02.02-.021.021-.021.021-.02a.387.387 0 00.02-.021l.021-.021.021-.021.021-.021.021-.021.021-.021.021-.021.021-.021.021-.021a.447.447 0 00.043-.043l.021-.021.021-.021.022-.021.021-.022.022-.021.021-.022a.447.447 0 01.022-.021l.021-.022a.447.447 0 01.022-.021.447.447 0 01.021-.022l.022-.022.022-.022.022-.021a.447.447 0 01.021-.022l.022-.022.022-.022.022-.022-1.414-1.414zM145 371l.707-.708-.002-.001-.008-.008-.032-.032-.126-.128a213.756 213.756 0 01-2.394-2.47 342.762 342.762 0 01-6.777-7.322c-5.611-6.235-13.091-14.961-20.568-24.931-7.48-9.973-14.942-21.17-20.53-32.347C89.672 291.859 86 280.773 86 271h-2c0 10.227 3.827 21.641 9.48 32.947 5.662 11.323 13.2 22.626 20.72 32.653 7.523 10.03 15.043 18.804 20.682 25.069 2.82 3.133 5.17 5.64 6.817 7.365a221.568 221.568 0 002.549 2.628l.033.033.009.009.002.002c.001.001.001.001.708-.706zM86 271c0-32.58 25.809-59 59-59v-2c-34.309 0-61 27.33-61 61h2zm59-59c32.573 0 59 26.427 59 59h2c0-33.677-27.323-61-61-61v2zm59 59c0 9.159-3.211 19.314-8.181 29.564-4.964 10.238-11.645 20.496-18.5 29.845l1.612 1.182c6.895-9.401 13.651-19.768 18.688-30.155C202.649 291.061 206 280.591 206 271h-2zm-26.699 59.433c-6.846 9.958-13.689 18.045-19.325 24.934l1.548 1.266c5.614-6.861 12.521-15.024 19.425-25.066l-1.648-1.134zM144.375 292c11.802 0 21-9.198 21-21h-2c0 10.698-8.302 19-19 19v2zm21-21c0-11.157-9.178-21-21-21v2c10.678 0 19 8.907 19 19h2zm-21-21c-11.177 0-21 9.823-21 21h2c0-10.073 8.927-19 19-19v-2zm-21 21c0 11.822 9.843 21 21 21v-2c-10.093 0-19-8.322-19-19h-2z" fill="#00B0B1"/><path opacity=".3" d="M277.344 221.023c-14.73 1.03-25.328 14.106-24.355 28.017 1.03 14.73 13.287 25.385 28.017 24.355l78.558-5.493c43.371-3.033 81.019 29.694 84.052 73.065 3.032 43.37-29.695 81.018-73.065 84.051l-226.673 15.851c6.85-7.88 14.289-19.091 22.489-31.176 4.395-7.709 9.608-15.474 13.946-24l186.576-13.047c13.911-.973 25.385-13.288 24.355-28.017-.973-13.912-14.106-25.328-28.018-24.355l-78.558 5.493c-44.189 3.09-81.837-29.637-84.87-73.007-3.032-43.371 29.695-81.019 73.884-84.109l34.369-2.403c-19.72-23.291-38.146-51.606-39.863-76.155-3.033-43.371 28.876-80.962 73.065-84.052 43.371-3.033 81.019 29.694 84.052 73.065 3.662 52.372-69.403 136.424-69.403 136.424l-78.558 5.493zm71.233-110.238c13.912-.972 25.385-13.287 24.355-28.017-.972-13.911-14.105-25.328-28.017-24.355-14.729 1.03-25.327 14.106-24.355 28.017 1.03 14.73 13.288 25.385 28.017 24.355zM125.32 422.431l-.761.875c-2.987 4.321-6.032 7.823-8.258 11.268-.818.057-.818.057-.761.875l-6.85 7.88s-84.051-73.065-87.713-125.437c-3.033-43.37 28.875-80.961 73.064-84.051 43.371-3.033 81.019 29.694 84.052 73.065 1.717 24.549-13.408 55.21-29.694 81.018-8.086 13.723-16.344 24.99-23.079 34.507zm-24.773-83.789c14.73-1.03 25.385-13.287 24.355-28.017-.972-13.911-13.287-25.385-28.017-24.355-13.911.973-25.328 14.106-24.355 28.017 1.03 14.73 14.106 25.328 28.017 24.355z" fill="#00B0B1"/></svg>
```

Comment:
Note: Could you pull the fill styling out of this element and move it to the Sass instead?

Reply:
Good shout!